lib/main_ltp.pm

```diff
@@ -62,14 +62,18 @@ sub load_kernel_tests {
         if (get_var('FLAVOR', '') =~ /Incidents-Kernel/) {
             loadtest_kernel 'update_kernel';
         }
-        if (is_transactional && (get_var('FLAVOR', '') =~ /-Staging|-Updates/)) {
-            if (get_var('KGRAFT')) {
-                loadtest_kernel 'update_kernel';
-            } else {
-                loadtest 'transactional/install_updates';
```

Comment:
Are you sure you want to completely drop `transactional/install_updates` from test schedule?

Reply:
My bad, thanks for catching missing else.

Reply:
This would be easier to understand:

    if ($needs_update && get_var('KGRAFT')) {
        loadtest_kernel 'update_kernel';
    }

    loadtest_kernel 'install_ltp';

    if ($needs_update && !get_var('KGRAFT')) {
        loadtest 'transactional/install_updates';
    }